lib/rules/web/images/jenkins_2/jenkins_2.xyaml

```diff
@@ -593,7 +593,7 @@ jenkins_oauth_login:
   action: jenkins_verify_logged_in
 
 jenkins_multi_oauth_login:
-  url: /securityRealm/commenceLogin?from=%2F
+  url: /
```

Comment:
@yapei  Could you help check ?  I'm not quite sure it will affect your team or not.

Reply:
I'll merge later today if still no response.